radixdlt/src/main/java/com/radixdlt/store/LedgerEntry.java

```diff
@@ -68,12 +76,13 @@ public boolean equals(Object o) {
 			return false;
 		}
 		LedgerEntry radixLedgerEntry = (LedgerEntry) o;
-		return aid.equals(radixLedgerEntry.aid);
+		return Objects.equals(aid, radixLedgerEntry.aid)
```

Comment:
See comment about content in hashCode.

Reply:
removed `content`, and ignoring it in EqualsVerifier test